hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashPolicyOzone.java

```diff
@@ -266,7 +266,8 @@ private void createCheckpoint(Path trashRoot, Date date) throws IOException {
     while (true) {
       try {
         fs.rename(current, checkpoint);
-        LOG.debug("Created trash checkpoint: " + checkpoint.toUri().getPath());
+        LOG.debug("Created trash checkpoint: {}",
+                checkpoint.toUri().getPath());
```

Comment:
this one is tricky, because URI.getPath() is not a O(1) operation. It constructs a String on the fly.
We need to wrap this debug message with if (LOG.isDebugEnabled()) {

Reply:
got it.